addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -92,6 +92,7 @@ const DEFAULT_HOOKS = {
 
 export function fetchRecordErrorHandler(env, error, originalError) {
     if (originalError instanceof FetchRecordError) {
+        error.event.preventDefault();
```

Comment:
maybe we should always prevent default in tests, even if the error service is deployed